resource/js/components/UserList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import UserPicture from './User/UserPicture';
+
+export default class UserList extends React.Component {
+
+  render() {
+    const users = this.props.users.map((user) => {
+      return (
+        <a data-user-id={user._id} href={'/user/' + user.username} title={user.name}>
+          <UserPicture user={user} size="xs" />
+        </a>
+      );
+    });
+
+    return (
+      <p className="seen-user-list">
```

Comment:
I think `UserList` can be used not only for `SeenUserList` but also for "Like" user list.
Where should such commonly usable components be placed?